service/db/src/test/java/gov/va/vro/service/db/SaveToDbServiceImplTest.java

```diff
@@ -40,71 +35,146 @@ class SaveToDbServiceImplTest {
+    assertEquals(claim.getVeteranIcn(), result.getVeteranIcn());
+    assertEquals(claim.getIncomingStatus(), result.getIncomingStatus());
+
+    assertEquals(1, veteranRepository.findAll().size());
+    assertEquals(1, claimRepository.findAll().size());
+    ClaimEntity claimEntity = claimRepository.findByVbmsId(claim.getBenefitClaimId()).orElseThrow();
+    assertEquals(claim.getBenefitClaimId(), claimEntity.getVbmsId());
+    assertEquals(claim.getVeteranIcn(), claimEntity.getVeteran().getIcn());
+    assertEquals(1, claimEntity.getContentions().size());
+    ContentionEntity contentionEntity = claimEntity.getContentions().get(0);
+    assertEquals(claim.getDiagnosticCode(), contentionEntity.getDiagnosticCode());
+    assertEquals(1, claimEntity.getClaimSubmissions().size());
+    ClaimSubmissionEntity claimSubmissionEntity =
+        claimEntity.getClaimSubmissions().iterator().next();
+    assertNotNull(claimSubmissionEntity);
+    assertEquals(claim.getCollectionId(), claimSubmissionEntity.getReferenceId());
+    assertEquals(claim.getIdType(), claimSubmissionEntity.getIdType());
   }
 
   @Test
   void persistAssessmentResult() throws Exception {
     // Save claim
     Claim claim = new Claim();
-    claim.setClaimSubmissionId("1234");
+    claim.setBenefitClaimId("1234");
+    claim.setIdType(MasAutomatedClaimPayload.CLAIM_V2_ID_TYPE);
     claim.setVeteranIcn("v1");
     claim.setDiagnosticCode("7101");
     saveToDbService.insertClaim(claim);
-    ClaimEntity claimBeforeAssessment =
-        claimRepository.findByClaimSubmissionId("1234").orElseThrow();
+    ClaimEntity claimBeforeAssessment = claimRepository.findByVbmsId("1234").orElseThrow();
     Map<String, Object> evidenceMap = new HashMap<>();
     evidenceMap.put("medicationsCount", "10");
     AbdEvidenceWithSummary evidence = new AbdEvidenceWithSummary();
+    evidence.setIdType(MasAutomatedClaimPayload.CLAIM_V2_ID_TYPE);
     evidence.setEvidenceSummary(evidenceMap);
+    // evidence flag is currently null
     saveToDbService.insertAssessmentResult(claimBeforeAssessment.getId(), evidence, "7101");
-    ClaimEntity result = claimRepository.findByClaimSubmissionId("1234").orElseThrow();
+    evidence.setSufficientForFastTracking(false);
+    saveToDbService.updateSufficientEvidenceFlag(evidence, "7101");
+    ClaimEntity result = claimRepository.findByVbmsId("1234").orElseThrow();
     assertNotNull(result);
     assertNotNull(result.getContentions().get(0).getAssessmentResults().get(0));
     AssessmentResultEntity assessmentResult =
         result.getContentions().get(0).getAssessmentResults().get(0);
     assertEquals(assessmentResult.getEvidenceCountSummary(), evidenceMap);
+    assertEquals(assessmentResult.getSufficientEvidenceFlag(), false);
 
     long c = assessmentResultRepository.count();
     assertEquals(1, c);
   }
 
+  @Test
+  void persistOffRampReason() {
+    Claim claim = new Claim();
+    claim.setBenefitClaimId("1234");
```

Comment:
For clarity:
```suggestion
    vbmsId="1234"
    claim.setBenefitClaimId("1234");
```
and change line 155 below to use the `vbmsId` variable.